resource_activity/models/sale.py

```diff
@@ -45,6 +45,19 @@ def action_confirm(self):
         if self.activity_sale and not self.env.context.get('activity_action'):
             raise UserError("You can't confirm a sale order linked to an activity. Please go to the activity to perform this operation")
         return super(SaleOrder, self).action_confirm()
+
+    @api.multi
+    def action_open_sale_order_line(self):

```

Comment:
change the function name if you display the sale orders and not the sale orders' lines

Reply:
Ok, it's a line in the tree but it's confusing, you're right.